python/pyspark/worker.py

```diff
@@ -128,6 +128,17 @@ def wrapped(*series):
     return lambda *a: (wrapped(*a), arrow_return_type)
 
 
+def wrap_window_agg_pandas_udf(f, return_type):
+    arrow_return_type = to_arrow_type(return_type)
+
+    def wrapped(*series):
+        import pandas as pd
+        result = f(*series)
+        return pd.Series([result]).repeat(len(series[0]))
```

Comment:
Just wondering why this needs to be repeated to the length of the series and grouped agg doesn't?

Reply:
Let's leave a short comment where we are here.

Reply:
Added comments to describe the function